File: auto-docs/executables/python/horizontal_histogram.py
import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')
import numpy as np
y = np.random.randn(500)

data = Data([
    Histogram(
        y=y
    )
])
plot_url = py.plot(data, filename='horizontal-histogram', auto_open=False)

File: auto-docs/executables/nodejs/horizontal_histogram.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var y = [];

for (var i = 0; i < 500; i ++) {
	y[i] = Math.random();
}

var data = [
  {
    y: y, 
    type: "histogram"
  }
];

var graph_options = {filename: "horizontal-histogram", fileopt: "overwrite", auto_open: false}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});